app/language/translations/en.yaml

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
Yes: map.yaml is not needed

Bill Cromie
t:917.215.3576

sent on the move

On Jan 23, 2015 7:26 PM, "Alex" <notifications@github.com> wrote:

> In app/language/translations/en.yaml
> <https://github.com/aberke/check-list/pull/6#discussion_r23490635>:
>
> > @@ -0,0 +1,76 @@
> > +---
>
> do the new en.yaml and es.yaml files make the map.yaml file unneeded?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aberke/check-list/pull/6/files#r23490635>.
>